standard/stream_links.lua

```diff
@@ -87,13 +103,25 @@ function StreamLinks.processStreams(forwardedInputArgs)
 		)
 
 		if String.isNotEmpty(streamValue) then
+			local platform = platformName
+			local languageCode = 'en'
+			local count = 1
 			-- stream has no platform
-			if platformName ~= 'stream' then
-				local lookUpPlatform = StreamLinks.streamPlatformLookupNames[platformName] or platformName
+			if platform ~= 'stream' then
+				if StreamLinks.streamPlatformLookupNames[platform] then
+					count = tonumber(platform:match('(%d+)$'))
+					platform = StreamLinks.streamPlatformLookupNames[platform]
+				end
+
+				streamValue = StreamLinks.resolve(platform, streamValue)
+			end
```

Comment:
This level of nesting is quite significant. It would be good to reduce it via a function

Reply:
`processStreams()` has been returned to its original form.